gui/gui/src/TGInputDialog.cxx

```diff
@@ -151,7 +151,8 @@ Bool_t TGInputDialog::ProcessMessage(Long_t msg, Long_t parm1, Long_t)
                   case 1:
                      // here copy the string from text buffer to return variable
                      // coverity[secure_coding]
-                     strcpy(fRetStr, fTE->GetBuffer()->GetString());
+                     str = fTE->GetBuffer()->GetString();
+                     strlcpy(fRetStr, str, sizeof(fRetStr));
```

Comment:
sizeof(fRetStr) does not work :( while it is just size of pointer (8 bytes).
Looking in the TGInputDialog constructor, we only can do:
```
if (fOwnBuf) strlcpy(fRetStr, str, 256);
             else strcpy(fRetStr, str); // unsafe, but we do not have size info
```

Reply:
Will revert and use `// NOLINT`